backend/app/routes/resources/@validationSchema/index.js

```diff
@@ -1,13 +1,27 @@
 const Joi = require('joi');
 
-const ResourcesValidationSchema = Joi.object().keys({
-  name: Joi.string().trim().required().min(3),
-  email: Joi.string().trim().email().required(),
-  url: Joi.string().trim().required().min(10),
-  description: Joi.string().trim().required().min(5),
-  trustLevel: Joi.number().integer().required(),
-  expiryDate: Joi.date().required(),
-  additionalInfo: Joi.string().trim().min(5),
+// const ResourcesValidationSchema = Joi.object().keys({
```

Comment:
please remove the commented code as it won't be needed anymore

Reply:
okay, also, we don't need resource model as we are using brodcast model and previous backend code addresource, get resource , deleteResource: should i remove it or skip that?

Reply:
Please remove it as it will just redundant code after this change

Reply:
![image](https://github.com/HITK-TECH-Community/Community-Website/assets/128178418/30e84113-20c4-4e8c-a9ce-6eaf20f7c8e0)


also removing it from the admin dashboard as we are managing all resources in admin dashboard

Reply:
yeah, that should be fine